README.md

```diff
@@ -211,6 +211,50 @@ catch (OperationFailedException e){
 ### Sample Application 
 For a more detailed example on how to use the library please take a look at our [Sample App](sample/).
 
+## Testing
+
+We use [ethereumjs/testrpc][testrpc] and [Truffle framework][truffle] unit tests.
```

Comment:
Links don't work - you need to change the path to be inside `(` parentheses
`(testrpc)`
`(truffle)`